classes/questions_form.php

```diff
@@ -256,7 +256,7 @@ public function definition() {
                 $manageqgroup[] =& $mform->createElement('image', 'editbutton['.$question->id.']', $esrc, $eextra);
                 $manageqgroup[] =& $mform->createElement('image', 'removebutton['.$question->id.']', $rsrc, $rextra);
 
-                if ($tid != QUESPAGEBREAK && $tid != QUESSECTIONTEXT) {
+                if ($tid != QUESPAGEBREAK && $tid != QUESSECTIONTEXT  && $tid != QUESSLIDER) {
```

Comment:
Why can't a slider question be required?

Reply:
Normally the slider question will have a default value, the user can answer or keep it, so the required condition here is not necessary.

Reply:
Okay. I will test this that way.